chipsec_util.py

```diff
@@ -124,40 +98,32 @@ def f_mod(self, x):
     def map_modname(self, x):
         return x.split('.')[0]
 
-
     def parse_args(self):
-        import getopt
-        try:
-            opts, args = getopt.getopt(self.argv, "ip:h:vdnl:", ["ignore_platform", "platform=", "help=", "verbose", "debug", "no_driver", "log=", "pch="])
-        except getopt.GetoptError, err:
-            logger().error(str(err))
-            self.chipsec_util_help()
-            sys.exit(ExitCode.EXCEPTION)
-        self._cmd_args = args
-        for o, a in opts:
-            if o in ("-v", "--verbose"):
-                logger().VERBOSE = True
-                logger().HAL     = True
-                logger().DEBUG   = True
-            elif o in ("-d", "--debug"):
-                logger().DEBUG   = True
-            elif o in ("-h", "--help"):
-                self.show_help = True
-                self.help_cmd  = a
-            elif o in ("-p", "--platform"):
-                self._platform = a.upper()
-            elif o in ("--pch"):
-                self._pch = a.upper()
-            elif o in ("-i", "--ignore_platform"):
-                self._unkownPlatform = False
-                logger().log( "[*] Ignoring unsupported platform warning and continue execution" )
-            elif o in ("-l", "--log"):
-                self.set_logfile(a)
-            elif o in ("-n", "--no_driver"):
-                self._no_driver = True
-            else:
-                pass
-
+        parser = argparse.ArgumentParser(prog='chipsec_util.py',usage='%(prog)s [options] <command>',add_help=False)
```

Comment:
Do we want to drop the .py from the prog paramter?

Reply:
Agree and changed